### ISSUE TITLE
improve accessability

### ISSUE THREAD

**Issue by SSilence** [SSilence/selfoss#1028]
I have some notes given by an blind user which works with an screen reader. Generally selfoss seems to be good for blind users. But some things could be improved:

- screen readers needs an alt text for favicons (title of the entry)
```
<!-- icon -->
    <a href="<?= \helpers\Anonymizer::anonymize($this->item['link']); 
?>" class="entry-icon" tabindex="-1">
        <?php if (strlen(trim($this->item['icon'])) > 0 && 
$this->item['icon'] != '0') : ?>
        <img src="<?= 'favicons/' . $this->item['icon']; ?>" alt="<?= 
$title; ?>" >
        <?php endif; ?>
    </a>
```
- There is no description on opened entrys
- stream button for more should be a clickable item (button or link) not just a div
- there should be an option that old entrys will disappear when new one loaded

**Comment by jtojnar** [comment]
1. The favicon alt was removed intentionally, since it is the same as item title and was read twice by screen readers during my testing. Since it has `tabindex="-1"`, it should be inert.
2. Hmm, I though I already set correct ARIA roles there. I will fix it.
3. I will do that after the client side refactoring.
4. Could they elaborate on that? I thought removing content is an accessibility issue. There are supposed to be some ARIA live regions, not sure how well they are supported.

**Comment by SSilence** [comment]
Thanks for the quick answer!! I will ask for details for the 4th note.

There is a 5th problem which seems to prevent blind users to use selfoss on a mobile device. The navigation can't be opened because the icon is not recognized. In home.phtml 
``<div id="nav-mobile-settings"></div>``
should be
``<div id="nav-mobile-settings"><button id="nav-mobile-settings" 
aria-label="<?= trim(\F3::get('lang_settingsbutton')); ?>"></button></div>``

Despite this bugs selfoss seems to be the best rss reader for blind users.

**Comment by jtojnar** [comment]
In #1133

1. I added `aria-hidden` attribute to make it even more hidden.
2. Added `aria-expanded` attribute but it might not be enough, future accessibility review will be necessary.
3. Done.
4. Not going to do that now, since I am not sure how to do that in accessible way.
5. Done.

**Comment by akash07k** [comment]
Great efforts @jtojnar 
I'll check all these changes and will try to fix somethings my self also. Along with that, will provide more detailed insights that what are the issues and how can they be fixed. Actually, I'm myself a blind user and working as Accessibility Subject matter expert.
I am profficient in development also, but I'm not familiar with selfoss codebase. So, I'll have to understand it.

**Comment by jtojnar** [comment]
Accessibility is never done but these particular issues should be finished now. Except for 4, which is tracked by #1135. Closing.
